test/js-native-api/test_general/test_general.c

```diff
@@ -1,3 +1,8 @@
+// we define this here to validate that it can
```

Comment:
nit: should we use `NODE_API_NO_EXTERNAL_BUFFERS_ALLOWED` here too?

Reply:
@vmoroz thanks for catching, fixed.